content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -191,6 +195,8 @@ kfctl apply -V -f ${CONFIG_FILE}
 
 * **${KF_DIR}** - The full path to your Kubeflow application directory.
 
+The Kubeflow endpoint is exposed with [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) 31380. If you don't have any experience on Kubernetes, you can [Expose the Kubeflow endpoint as LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer) and access the **EXTERNAL_IP**.
```

Comment:
nit:

```suggestion
The Kubeflow endpoint is exposed with [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) 31380. If you don't have any experience on Kubernetes, you can [expose the Kubeflow endpoint as LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer) and access the **EXTERNAL_IP**.
```

It may make sense to add a link to LoadBalancer somewhere too, I think: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/